bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/handler/OpenThermGatewayHandler.java

```diff
@@ -52,38 +42,35 @@
  * @author Arjen Korevaar - Initial contribution
  */
 @NonNullByDefault
-public class OpenThermGatewayHandler extends BaseThingHandler implements OpenThermGatewayCallback {
+public class OpenThermGatewayHandler extends BaseBridgeHandler implements OpenThermGatewayCallback {
 
     private final Logger logger = LoggerFactory.getLogger(OpenThermGatewayHandler.class);
 
-    private @Nullable OpenThermGatewayConfiguration config;
+    private @Nullable OpenThermGatewayConfiguration configuration;
     private @Nullable OpenThermGatewayConnector connector;
     private @Nullable ScheduledFuture<?> reconnectTask;
 
-    private boolean connecting = false;
-    private boolean explicitDisconnect = false;
+    private @Nullable ConnectionState state;
+    private boolean autoReconnect = true;
 
-    public OpenThermGatewayHandler(Thing thing) {
-        super(thing);
+    public OpenThermGatewayHandler(Bridge bridge) {
+        super(bridge);
     }
 
     @Override
     public void initialize() {
-        logger.debug("Initializing OpenTherm Gateway handler for uid '{}'", getThing().getUID());
+        logger.debug("Initializing OpenThermGateway handler for uid {}", getThing().getUID());
 
-        updateStatus(ThingStatus.UNKNOWN, ThingStatusDetail.NONE, "Initializing");
-
-        config = getConfigAs(OpenThermGatewayConfiguration.class);
+        configuration = getConfigAs(OpenThermGatewayConfiguration.class);
+        logger.info("Using configuration: {}", configuration.toString());
```

Comment:
Can this message be replaced by using the debugger? See point 4 https://www.openhab.org/docs/developer/guidelines.html#f-logging